Remotion/Core/Development.Analyzers.IntegrationTests/RMCORE0001_SafeContextAnalyzerTests.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using System.Threading.Tasks;
+using Core.Development.Analyzers;
+using NUnit.Framework;
+using Verifier = Remotion.Core.Development.Analyzers.IntegrationTests.CSharpAnalyzerVerifier<Core.Development.Analyzers.RMCORE0001_SafeContextAnalyzer>;
+using Analyzer = Core.Development.Analyzers.RMCORE0001_SafeContextAnalyzer;
+
+namespace Remotion.Core.Development.Analyzers.IntegrationTests
+{
+  [TestFixture]
+  public class RMCORE0001_SafeContextAnalyzerTests
```

Comment:
There are a few formatting problems in this file,. Should be fixed with a reformat command.

Reply:
Not sure about that, reformatting only removed the `withSafeContextReference` parameter description